src/overlays/actors/ovl_En_Invadepoh_Demo/z_en_invadepoh_demo.c

```diff
@@ -35,134 +105,706 @@ ActorInit En_Invadepoh_Demo_InitVars = {
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Invadepoh_Demo/func_80C19564.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Invadepoh_Demo/func_80C19590.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Invadepoh_Demo/func_80C1965C.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Invadepoh_Demo/func_80C19688.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Invadepoh_Demo/func_80C19754.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Invadepoh_Demo/func_80C1985C.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Invadepoh_Demo/func_80C1987C.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Invadepoh_Demo/func_80C199BC.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Invadepoh_Demo/func_80C199EC.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Invadepoh_Demo/func_80C19AB4.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Invadepoh_Demo/func_80C19D00.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Invadepoh_Demo/func_80C19D48.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Invadepoh_Demo/func_80C19E04.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Invadepoh_Demo/func_80C19EC0.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Invadepoh_Demo/func_80C19F7C.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Invadepoh_Demo/func_80C1A038.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Invadepoh_Demo/func_80C1A070.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Invadepoh_Demo/func_80C1A168.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Invadepoh_Demo/func_80C1A244.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Invadepoh_Demo/func_80C1A590.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Invadepoh_Demo/func_80C1A650.s")
+static EnInvadepohDemoFunc sInitFuncs[EN_INVADEPOH_DEMO_TYPE_MAX] = {
+    EnInvadepohDemo_Alien_Init,   // EN_INVADEPOH_DEMO_TYPE_ALIEN
+    EnInvadepohDemo_Romani_Init,  // EN_INVADEPOH_DEMO_TYPE_ROMANI
+    EnInvadepohDemo_Cow_Init,     // EN_INVADEPOH_DEMO_TYPE_COW
+    EnInvadepohDemo_Ufo_Init,     // EN_INVADEPOH_DEMO_TYPE_UFO
+    EnInvadepohDemo_CowTail_Init, // EN_INVADEPOH_DEMO_TYPE_COW_TAIL
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Invadepoh_Demo/func_80C1A670.s")
+static EnInvadepohDemoFunc sDestroyFuncs[EN_INVADEPOH_DEMO_TYPE_MAX] = {
+    EnInvadepohDemo_DoNothing,       // EN_INVADEPOH_DEMO_TYPE_ALIEN
+    EnInvadepohDemo_DoNothing,       // EN_INVADEPOH_DEMO_TYPE_ROMANI
+    EnInvadepohDemo_Cow_Destroy,     // EN_INVADEPOH_DEMO_TYPE_COW
+    EnInvadepohDemo_DoNothing,       // EN_INVADEPOH_DEMO_TYPE_UFO
+    EnInvadepohDemo_CowTail_Destroy, // EN_INVADEPOH_DEMO_TYPE_COW_TAIL
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Invadepoh_Demo/func_80C1A6C8.s")
+static EnInvadepohDemoFunc sAlienCueActions[EN_INVADEPOH_DEMO_ALIEN_CUE_ID_MAX] = {
```

Comment:
It'd be nice if `sAlienCueActions` had `Func` in its name, but I can see why `sAlienCueActionFuncs` wouldn't work since these aren't action functions, but action is the consistent word we've been using in the context of cutscenes. Maybe `sAlienCueActionCsFuncs`?